### ISSUE TITLE
Explore fun new design concepts / branding for Tor

### ISSUE THREAD

**Issue by bnvk** [opensourcedesign/logos#2]
As per this neat start by @mcginty which I really like this start as an improvement over the existing Tor logo / branding- some ideas to explore a few variations (which I'd be also curious to see) are:

![Tor logo icon v1](https://raw.githubusercontent.com/opensourcedesign/logos/master/unofficial/tor/tor_icon.png)
- Make the thickness of the T and R a thicker weight font (extra / ultra)
- Make the centric rings slightly thicker, currently will lose details scaled down
- Make middle dot a little larger in size, like 20% bigger
- Add a few more dots (like the center one) placed "randomly / staggered" on the outwards rings
- Loose the faint drop shadow

**Comment by evalica** [comment]
I guess this thread is about https://www.torproject.org/ 
My first reaction was to think of the https://onionproject.org/ since they presented their project on the FOSDEM "Developers looking for designers show off your project" presentation. So if we produce multiple variations maybe they will be interested in selecting one from the remainders.

**Comment by mcginty** [comment]
Yep, this is focused on Tor Project iconography. The goal with having a liberally licensed asset with sources will be that if a new icon pops up that's ready for larger adoption, it can be easily modified and re-used for subprojects.

**Comment by bnvk** [comment]
@evalica ah, yah there are numerous projects that put that call themselves "OnionSomething" or alternatively "OrSomething" so as to not conflict with Tor's brand identity + official apps such as:
- [OnionShare](https://onionshare.org)
- [OrBot](https://guardianproject.info/apps/orbot)

I suggested Jake publish these fun "unofficial" logos for Tor (the organization) to share + get feedback about his designs!

@mcginty like Eva said, you could also re-purpose / modify this for OnionShare, which currently has a slightly lacking logo... maybe you'd want to add some icon relating to "sharing" to your design, and make the icon be the "O" of NIONSHARE
